app/dao/organisation_dao.py

```diff
@@ -246,3 +246,13 @@ def dao_add_letter_branding_list_to_organisation_pool(organisation_id, letter_br
     organisation.letter_branding_pool.extend(letter_brandings)
 
     db.session.add(organisation)
+
+
+@autocommit
```

Comment:
Does this immediately cause a commit to happen? Or are all of our requests wrapped in outer transactions that mean errors later in the view would stop this being committed?

Reply:
It will cause a [commit to happen (or a rollback if there is an error)](https://github.com/alphagov/notifications-api/blob/b2aeed613d1185c146414b75e2d8fce97f527ddb/app/dao/dao_utils.py#L9) when the dao function has finished executing all its code.

Reply:
🤔 🤔 🤔